indexer/base/kind.go

```diff
@@ -66,17 +72,271 @@ const (
 	// Float64Kind is a float64 type and values of this type must be of the go type float64.
 	Float64Kind
 
-	// Bech32AddressKind is a bech32 address type and values of this type must be of the go type string or []byte
-	// or a type which implements fmt.Stringer. Fields of this type are expected to set the AddressPrefix field
-	// in the field definition to the bech32 address prefix.
+	// Bech32AddressKind is a bech32 address type and values of this type must be of the go type []byte.
```

Comment:
I switched the requirement to be just `[]byte`s so the validation logic doesn't need to do any bech32 parsing here (although it is a pretty lightweight dependency). Also wondering if we need a length limit on addresses or not.

Reply:
How about a length limit of 64 or 128 bytes for addresses?